scope3_methodology/adtech_model.py

```diff
@@ -74,12 +73,12 @@ def getProductInfo(key: str, default: float | None, product: dict[str, float], d
 
 
 def getCorporateEmissionsPerBidRequest(
+    corporateEmissionsG: float,
```

Comment:
why not snake case?

Reply:
I am so used to js. 🙈  I will fix this

Reply:
We are also super not consistent in the code